run_thread_switcher_prime95.bat

```diff
@@ -0,0 +1,3 @@
+cp settings_prime95.py settings.py
```

Comment:
Love this, being able to use two separate settings files by overwriting a common location, perfect solution!

Reply:
```suggestion
copy settings_prime95.py settings.py
```

Reply:
Thanks!

I did this and added `/Y` as well as `copy` doesn't overwrite by default.